cmd/atlasci/internal/ci/ci.go

```diff
@@ -242,18 +243,18 @@ func (d *DevLoader) LoadChanges(ctx context.Context, files []migrate.File) ([]*s
 				return nil, err
 			}
 			current = target
-			// In case the change is recognized by Atlas.
-			if len(changes) > 0 {
```

Comment:
Because I want to keep statements like `INSERT INTO ..` in the file. e.g. detect cases like `CREATE`, `INSERT`, `RENAME` and `DROP` as a table modification change in SQLite.